src/testing/builderbot/simulator/example_builderbot_advance.lua.in

```diff
@@ -0,0 +1,86 @@
+assert(loadfile("@CMAKE_INSTALL_PREFIX@/include/argos3/plugins/robots/builderbot/lua_library.luac"))()
+
+--local data -- make data global so that it can be seen in lua_editor
+local behavior
```

Comment:
Was this for debugging or should it be left like this?

Reply:
same above

Reply:
As above